components/blocks/FilesToCards.vue

```diff
@@ -10,8 +10,7 @@
         card-container
         is-flex
         mt-6
-        is-justify-content-space-evenly 
-        is-flex-wrap-wrap
+        is-justify-content-space-evenly is-flex-wrap-wrap
```

Comment:
did the linter do this? it looks kind of funny to have most classes one per line, then this one two per line